docs/book/developer-guide/materializer.md

```diff
@@ -124,7 +117,9 @@ For more information, visit https://docs.zenml.io/guides/common-usecases/custom-
 ```
 
 The above basically means that ZenML does not know how to persist the object of type `MyObj` between steps (how could 
-it? We just created this!). Therefore, we can create our own materializer:
+it? We just created this!). Therefore, we can have to create our own materializer. To do this you can simply extend the 
```

Comment:
I can't comment on the error message above, but that contains a wrong URL to our docs. We will need to fix this both in code and here to point to the updated version.

Reply:
Definitely, catching outdated urls is gonna be quite the task